Sources/ArcGISToolkit/Common/AttachmentModel.swift

```diff
@@ -90,25 +90,8 @@ import SwiftUI
                 systemImageName = "exclamationmark.circle.fill"
                 return
             }
-            var url = attachment.fileURL!
-            if attachment is FormAttachment {
-//                self.thumbnail = try? await attachment.makeThumbnail(
-//                    width: Int(thumbnailSize.width),
-//                    height: Int(thumbnailSize.width)
-//                )
-                
-                // WORKAROUND - attachment.fileURL is just a GUID for FormAttachments
-                // Note: this can be deleted when Apollo #635 - "FormAttachment.fileURL is not user-friendly" is fixed.
-                var tmpURL = attachment.fileURL
-                tmpURL = tmpURL?.deletingLastPathComponent()
-                tmpURL = tmpURL?.appending(path: attachment.name)
-                
-                _ = FileManager.default.secureCopyItem(at: attachment.fileURL!, to: tmpURL!)
-                url = tmpURL!
-            }
```

Comment:
Hmm, I thought I already pulled these out in [715](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/715/files). Maybe this branch was just behind? Not sure why the change is still showing as a diff though.

Reply:
I merged the `FeatureFormAttachments` branch into this branch and that went away. Sorry I didn't do that first.

Reply:
Ah nice, no worries it just had me confused for a moment.